uvicorn/config.py

```diff
@@ -57,12 +57,12 @@
 SSL_PROTOCOL_VERSION = getattr(ssl, "PROTOCOL_TLS", ssl.PROTOCOL_SSLv23)
 
 
-LOGGING_CONFIG = {
+LOGGING_CONFIG_COLORS = {
```

Comment:
Let's just stick with a single `LOGGING_CONFIG`.
We can pass an argument `"use_colors": False,` in the config to the `DefaultFormatter`.

Reply:
done